docs/manage/fleet/_index.md

```diff
@@ -3,51 +3,52 @@ title: "Fleet Management"
 linkTitle: "Fleet Management"
 weight: 30
 type: "docs"
-description: "Configure, control, debug, and manage your robots from the cloud at app.viam.com."
+description: "Configure, control, debug, and manage your robots from the cloud at app.viam.com on your own or with a team."
 tags: ["fleet management", "cloud", "app"]
+no_list: true
 aliases:
     - "/manage/fleet-management"
     - "/manage/app-usage"
     - "/product-overviews/fleet-management/"
 ---
 
-The [Viam app](https://app.viam.com) provides fleet management allowing you to:
+The [Viam app](https://app.viam.com) provides fleet management allowing you to work alone or in collaboration with others on one or more robots.
 
-- manage your robots and access to them with [locations](#locations), [organizations](#organization), and [permissions](#permissions)
-- [configure](#configuration) individual or groups of robots
-- [deploy code and machine learning models](#package-deployment) to robots
-- [remotely control](#remote-control) and [debug](#logs) robots
+## Work with groups of robots
 
-All communication happens securely over HTTPS using secret tokens that are in a robot's configuration.
+With Viam, you can organize {{< glossary_tooltip term_id="robot" text="robots" >}} into {{< glossary_tooltip term_id="location" text="locations" >}} and {{< glossary_tooltip term_id="organization" text="organizations" >}}.
 
-## Robots
+For example, you may have separate organizations for your robots at home and at work:
 
-An organizational concept, consisting of either one _{{< glossary_tooltip term_id="part" text="part" >}}_, or multiple _parts_ working closely together to complete tasks.
+{{<gif webm_src="../img/organizations.webm" mp4_src="../img/organizations.mp4" alt="An organization for personal robots and one for work robots.">}}
 
-For more information on robots and how to manage them, see [Robots](robots).
+Inside an organization, you can organize robots into one or more locations:
 
-## Locations
+![An image of two locations, New York, and Chicago, in one organization, Good Robots](../img/locations.png)
 
-In Viam, every robot belongs to a location.
-A location is a virtual grouping of robots that allows you to organize robots and manage access.
+If you are managing a large fleet, you can use {{< glossary_tooltip term_id="fragment" text="fragments" >}} when [configuring your robots](../configuration), allowing you to use the same configuration for multiple robots.
 
-![An image of two locations, New York, and Chicago, in one organization, Good Robots](../img/locations.png)
+## Use Viam for collaboration
 
-For information on locations and how to manage them, see [Locations](locations).
+To facilitate collaboration, you can add collaborators to organizations, and share locations across multiple organizations. Soon you will also be able to assign [permissions](#permissions) to collaborators.
```

Comment:
```suggestion
To facilitate collaboration, you can add collaborators to organizations, and share locations across multiple organizations.
Soon you will also be able to assign [permissions](#permissions) to collaborators.
```
Do we want to talk about currently unsupported things? Or just add them once we support them?

Reply:
Since this will happen in the next week or two I don't mind it